Library/templates/admin_layout.html

```diff
@@ -16,7 +16,9 @@
 </head>
 
 <body
-     style="font-family: 'PT Sans', sans-serif;background-image: url(https://www.pixelstalk.net/wp-content/uploads/2016/08/Library-Desktop-Wallpaper-768x480.jpg);background-size: cover;">
+     style="font-family: 'PT Sans', sans-serif;">
+     <!-- background-image: url(https://www.pixelstalk.net/wp-content/uploads/2016/08/Library-Desktop-Wallpaper-768x480.jpg) -->

```

Comment:
You can better remove the background image

Reply:
It looked nice without the background image